cli/command_snapshot_create.go

```diff
@@ -336,7 +349,7 @@ func (c *commandSnapshotCreate) snapshotSingleSource(ctx context.Context, fsEntr
 		}
 	}
 
-	c.svc.getProgress().Finish()
+	progress.Finish()
```

Comment:
not totally sure if I should first `Finish()` before I access the values in `progress`. to me it seems like that the `Finish` function only changes what is displayed in the cli but doesn't affect the integer values, thus I think it doesn't matter when `Finish` is called